### PR TITLE
adding swift string operations

### LINKED ISSUE #388: Add Swift strings
# Description

Strings for Swift aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/strings.json` to the right Swift directory in `web/thesauruses/`
* Edit the strings.json file to accurately reflect the latest major version of Swift
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then delete the `"code"` line.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by samuelburns**
## What GitHub issue does this PR apply to?

Resolves #388

## What changed and why?

Added swift string operations

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [ ] I have looked at documentation to ensure I made any revisions correctly
- [X] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/swift/strings.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+      "format_number_in_octal",
+      "format_number_in_hexadecimal"
+    ]
+  },
+  "strings": {
+    "is_primitive_or_not": {
+      "code": "Yes"
+    },
+    "import": {
+      "code": "Not required"
```
Comment (geekygirlsarah) [inline web/thesauruses/swift/strings.json]: This can probably just turn into `"not-implemented": true" instead of the code line.

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
If you change the one line, I'll merge it in!

File: web/thesauruses/swift/strings.json (hunk 1 above)
Comment (samuelburns) [reply web/thesauruses/swift/strings.json]: Done, thanks for spotting this

**Comment by geekygirlsarah** [comment]
The update looks good to me. Thanks for fixing that. I'll merge it in!

**Pull request merged**